vercel.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
also unrelated?

Reply:
Will remove - was just for my testing on vercel